enterprise/models.py

```diff
@@ -1481,7 +1481,11 @@ def enrollments_for_user(cls, enterprise_customer_user):
         """
         return cls.objects.filter(
             enterprise_course_enrollment__enterprise_customer_user=enterprise_customer_user
-        ).select_related('enterprise_course_enrollment')
+        ).select_related(
+            'enterprise_course_enrollment',
+            'enterprise_course_enrollment__enterprise_customer_user',
+            'enterprise_course_enrollment__enterprise_customer_user__enterprise_customer',
```

Comment:
Update the doc comment to include these. Also do we need to return these in this function?

Reply:
Ah, yeah, we need those because this thing's caller looks at each of those associated model instances, so better to get it out of the way with a single query.